web-wallet/src/routes/(app)/dashboard/staking/__tests__/page.spec.js

```diff
@@ -0,0 +1,17 @@
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { cleanup, render } from "@testing-library/svelte";
+import Staking from "../+page.svelte";
+
+vi.useFakeTimers();
+
+describe("Staking", () => {
+  afterEach(cleanup);
+
+  it("should render the staking page", async () => {
+    const { container } = render(Staking);
+
+    await vi.advanceTimersToNextTimerAsync();
```

Comment:
If we need the await for the basic test, maybe it's better to add a comment about it?
But do we need it? Can this just be without fake timers at all?

Reply:
Without this, the stakeInfo promise is not resolved in the Suspense component making the snapshot render the Throbber.